dell-restore-system

```diff
@@ -40,6 +40,8 @@ parser.add_option('-c', '--check-version', dest='checkversion', action='store_tr
                   help='Show the version information.')
 parser.add_option('-y', '--yes', dest='yes', action='store_true',
                   help="Restore the system without user's confirmation.")
+parser.add_option('-d', '--dhc_flag', dest='dhc_flag', action='store_true',
```

Comment:
--dhc_flag is a little confusing with the - and _ shift, can you make --dhc or --dhc-flag instead?